robot_smach_states/src/robot_smach_states/manipulation/active_grasp_detector.py

```diff
@@ -14,7 +14,7 @@ class ActiveGraspDetector(smach.State):
     REQUIRED_ARM_PROPERTIES = {"required_gripper_types": [GripperTypes.GRASPING], }
 
     def __init__(self, robot: Robot, arm_designator: ArmDesignator, threshold_difference: float = 0.075,
-                 minimum_position: float = -0.82, max_torque: float = 0.15) -> None:
+                 minimum_position: float = -0.75, max_torque: float = 0.15) -> None:
```

Comment:
These are still very much magic numbers. They have the downside that they need to be tuned constantly. But I think that may be for another time

Reply:
Make this an issue. If we need keep changing this, this is something we need to work on with high priority.

Reply:
I created an issue to look into the grasp position detector robot part https://github.com/tue-robotics/tue_robocup/issues/1248 . It does not have high priority though. So I don't think we need to add a deadline